CHANGELOG.md

```diff
@@ -4,6 +4,25 @@ All notable changes to secrets-store-csi-driver-provider-gcp will be documented
 
 ## unreleased
 
+## v1.3.0
+
+Images:
+
+* `asia-docker.pkg.dev/secretmanager-csi/secrets-store-csi-driver-provider-gcp/plugin:v1.3.0`
+* `europe-docker.pkg.dev/secretmanager-csi/secrets-store-csi-driver-provider-gcp/plugin:v1.3.0`
+* `us-docker.pkg.dev/secretmanager-csi/secrets-store-csi-driver-provider-gcp/plugin:v1.3.0`
+
+Digest: `sha256:af3dde5fca31b17599d71cd5ec63b52289a1efcee26d0fa4205e9da39a5f53f4`
+
+
+### Changes & Security Fixes
+
+* Prevent the sharing of any host namespaces [#289](https://github.com/GoogleCloudPlatform/secrets-store-csi-driver-provider-gcp/pull/289)
+* Modify Helm DaemonSet to prevent the sharing of host namespaces [#290](https://github.com/GoogleCloudPlatform/secrets-store-csi-driver-provider-gcp/pull/290)
+* Make filesystem read only [#292](https://github.com/GoogleCloudPlatform/secrets-store-csi-driver-provider-gcp/pull/292)
+* Dependency Updates [#294](https://github.com/GoogleCloudPlatform/secrets-store-csi-driver-provider-gcp/pull/294)
```

Comment:
Please update change log to include commits related to arm64 toleration and sustainable solution for using workload identitiy